.circleci/config.yml

```diff
@@ -472,6 +484,21 @@ jobs:
             - node_modules
             - build-artifacts
 
+  get-changed-files-with-git-diff:
```

Comment:
Nit: I see that this is now expected to be run before `run-all.js`. But this dependency may not be clear for readers. perhaps we could add a comment here, or somewhere, to explain that this must be a requirement of all E2E test runs